openbb_terminal/keys_controller.py

```diff
@@ -137,6 +146,44 @@ def call_mykeys(self, other_args: List[str]):
                 else None,
             )
 
+    @log_start_end(log=logger)
+    def call_status(self, other_args: List[str]):
+        """Display current keys status with the option to force the check"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="status",
+            description="Reevaluate keys status.",
+        )
+        parser.add_argument(
+            "-k",
+            "--key",
+            type=str,
+            dest="key",
+            help="Force the status to reevaluate for a specific key.",
+            default=None,
+        )
+
+        if other_args and "-" not in other_args[0][0]:
+            other_args.insert(0, "-k")
+
+        ns_parser = self.parse_known_args_and_warn(
+            parser, other_args, export_allowed=EXPORT_ONLY_RAW_DATA_ALLOWED
+        )
+
+        is_key = bool(ns_parser and ns_parser.key)
+
+        if is_key:
+            if ns_parser.key in self.API_LIST:
+                self.status_dict[ns_parser.key] = self.get_check_keys_function(
```

Comment:
@montezdesousa should this be in a try except?

Reply:
Yes, some check functions don't handle that properly